config/defaults.mk

```diff
@@ -473,7 +473,17 @@ OCCA_LIB = $(XLINKER)-rpath,$(OCCA_DIR)/lib -L$(OCCA_DIR)/lib -locca
 # CALIPER library configuration
 CALIPER_DIR = @MFEM_DIR@/../caliper
 CALIPER_OPT = -I$(CALIPER_DIR)/include
-CALIPER_LIB = $(XLINKER)-rpath,$(CALIPER_DIR)/lib64 -L$(CALIPER_DIR)/lib64 -lcaliper
+CALIPER_LIB = $(XLINKER)-rpath,$(CALIPER_DIR)/lib -L$(CALIPER_DIR)/lib -lcaliper
```

Comment:
We can support both `lib` and `lib64` by listing them both:
```make
CALIPER_LIB = $(XLINKER)-rpath,$(CALIPER_DIR)/lib64 $(XLINKER)-rpath,$(CALIPER_DIR)/lib\
 -L$(CALIPER_DIR)/lib64 -L$(CALIPER_DIR)/lib -lcaliper
```
We do this for some other libs (e.g. SUNDIIALS) -- it allows us to avoid checking the filesystem to determine the right library dir.

Reply:
done